src/main/java/de/tum/in/www1/artemis/web/rest/ApollonDiagramResource.java

```diff
@@ -1,6 +1,6 @@
 package de.tum.in.www1.artemis.web.rest;
 
-import static de.tum.in.www1.artemis.web.rest.util.ResponseUtil.conflict;
+import static de.tum.in.www1.artemis.web.rest.util.ResponseUtil.*;
```

Comment:
Are these changes on purpose? I thought we wanted to avoid star imports. (Note: This happened on multiple files)

Reply:
I wanted to decrease the number of warnings for now, importing deprecated static methods counts as a warning, but does not really help fixing those or having an overview how many issues are left